server/gitlab/api.go

```diff
@@ -258,24 +298,50 @@ func (g *gitlab) GetYourPrs(user *UserInfo) ([]*internGitlab.MergeRequest, error
 	opened := stateOpened
 	scope := scopeAll
 
-	var result []*internGitlab.MergeRequest
+	var prList []*internGitlab.MergeRequest
 	var errRequest error
+	var resp *internGitlab.Response
+	var result []*internGitlab.MergeRequest
```

Comment:
`result` is usually used for the "final result", but it's being used as an intermediate value below. Maybe use `current` and `result` to convey the meaning of each variable